sechub-scan/src/test/java/com/mercedesbenz/sechub/domain/scan/AbstractInstallSetupTest.java

```diff
@@ -26,7 +26,7 @@ public void isabletoscan_returns_false_when_a_target_has_type_intranet_and_not_a
     @Test
     public void isabletoscan_returns_true_when_a_target_has_type_intranet_and_is_able_to_scanDaimmler_intranet() {
         /* prepare */
-        setupToTest.canScanDaimlerIntranet = true;
+        setupToTest.canScanIntranet = true;
```

Comment:
Good change: You made it neutral :+1:

Reply:
@sven-dmlr : ups ... i still missed the method name, contains:`Daimmler` will change this as well and merge than